docs/updates/2020-08-07-announce-book.md

```diff
@@ -0,0 +1,108 @@
+# 2020-08-07 - Announcing the new Jupyter Book
+
+*Note: this announcement is cross-posted between the [Jupyter Blog](https://blog.jupyter.org) and the [Executable Book Project updates blog](https://executablebooks.org/en/latest/updates/index.html)*
+
+Jupyter Book is an open source project for building beautiful, publication-quality books and documents from computational material. With this post, we're happy to announce that Jupyter Book has been re-written from the ground up, making it easier to install, faster to use, and able to create more complex publishing content in your books. It is now supported by [the Executable Book Project](https://executablebooks.org/en/latest/), an open community that builds open source tools interactive and executable documents in the Jupyter ecosystem and beyond.
+
+```{figure} images/jb-ui.png
+
+The new Jupyter Book interface, highlighting how you can [insert code outputs into your content](https://jupyterbook.org/content/glue.html)
+```
+
+This post is a short overview of the new features in Jupyter Book, and gives some background on the future of the project.
+
+You can also browse the [new Jupyter Book documentation](https://jupyterbook.org), or [check out the Jupyter Book GitHub repository](https://github.com/executablebooks/jupyter-book).
+
+**Note**: Jupyter Book is still in beta, so things are always changing. We'd love your feedback, ideas, and PRs for how to make the project better!
```

Comment:
````suggestion
```{note}
Jupyter Book is still in beta, so things are always changing. We'd love your feedback, ideas, and PRs for how to make the project better!
```
````